app/src/main.ts

```diff
@@ -1,29 +1,29 @@
 import 'source-map-support/register';
+
 import fs from 'fs';
 import path from 'path';
+
 import { app, crashReporter, globalShortcut } from 'electron';
 import electronDownload from 'electron-dl';
 
-import createLoginWindow from './components/login/loginWindow';
-import createMainWindow from './components/mainWindow/mainWindow';
-import createTrayIcon from './components/trayIcon/trayIcon';
-import helpers from './helpers/helpers';
-import inferFlash from './helpers/inferFlash';
+import { createLoginWindow } from './components/loginWindow';
+import { createMainWindow } from './components/mainWindow';
+import { createTrayIcon } from './components/trayIcon';
+import * as helpers from './helpers/helpers';
+import { inferFlashPath } from './helpers/inferFlash';
 
-const electronSquirrelStartup = require('electron-squirrel-startup');
-
-// Entrypoint for electron-squirrel-startup.
-// See https://github.com/jiahaog/nativefier/pull/744 for sample use case
-if (electronSquirrelStartup) {
-  app.exit();
-}
+// Entrypoint for Squirrel, a windows update framework. See https://github.com/jiahaog/nativefier/pull/744
+// TODO disabled for now, breaks build: Uncaught Exception: Error: Cannot find module '/tmp/app-linux-x64/resources/app/node_modules/electron-squirrel-startup/node_modules/debug/src/index.js'. Please verify that the package.json has a valid "main" entry
```

Comment:
> Should probably include this in the PR description if this is disabled in case people are relying on this

@jiahaog I disabled it as I was trying to advance elsewhere; will repair it. I precised the associated TODO entry:

```
- [ ] Address TODOs in code, especially repair Squirrel
```